lib/ibmdb.js

```diff
@@ -314,14 +314,31 @@ function dateToIBMDB(val) {
   }
 };
 
+IBMDB.prototype.toColumnValue = function(prop, val) {
+  debug('IBMDB.prototype.toColumnValue prop=%j val=%j', prop, val);
+  const transformedValue = this.transformColumnValue(prop, val);
+  if (val == null || !prop || !prop.db2) {
+    return transformedValue;
+  }
+  // db2.datatype needs to be defined in User Defined Model Definition
+  switch (prop.db2.dataType) {
+    case 'BLOB':
+      return {CType: 'BLOB', DataType: 'BLOB', Data: transformedValue};
+    case 'CLOB':
+      return {DataType: 'CLOB', Data: transformedValue};
```

Comment:
Should there be a `CType` here as well?

Reply:
@virkt25 @dhmlau @sam-github @rmg : please let me know what is needed to merge this PR.

Reply:
/cc @bajtos